src/main/java/frc/robot/subsystems/CDSSubsystem.java

```diff
@@ -32,28 +46,41 @@ public CDSSubsystem() {
   public void CDSBeltWheelControllerToggle(boolean reverse) {
     if (reverse) {
       CDSWheelControllerOne.getSparkMax().set(Constants.CDSWheelControllerSpeed);
-      SmartDashboard.putString("CDS Wheel Controller Direction", "Reverse");
-      SmartDashboard.putNumber("CDS Wheel Controller Speed", -Constants.CDSWheelControllerSpeed);
```

Comment:
Why did we remove publishing the CDS wheel speed to the SmartDashboard? The intake team still uses this for debugging

Reply:
Cause it is replaced by the NetworkTableEntry which would put the entire thing onto a different tab for visibility

Reply:
There is a Network Table entry for the direction, but I don't see one for the speed value?